style.css

```diff
@@ -0,0 +1,85 @@
+#header {
+  display: flex;
+  background-color: #3c3a39;
+  max-width: 375px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+#owner_name p {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
+}
+
+#owner_name {
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+#owner_name a {
+  text-decoration: none;
+}
+
+#header_menu,
+#owner_name {
+  padding: 0 1rem;
+}
+
+#headline {
+  background: #1c1a19;
+  max-width: 375px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-bottom: 100px;
+  background-image: url(/images/Illustration.png);
```

Comment:
- [ ] Good job using the background image in the CSS file. I think there is some issue in the path and the image is not getting rendered correctly. Try adding a `.` before the path so it will be displayed. 🤝️

```
background-image: url(./images/Illustration.png);
```

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/211836013-d552e48d-289e-4d42-b71f-2af725a3ba89.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/211836026-09e9f83c-5e66-4bc5-92b5-cc3bfa067503.png" width="300">|

- [ ] [OPTIONAL] I highly suggest you add the necessary fonts to match the design. You can use [Google Fonts](https://fonts.google.com/) to grab the required fonts and add them to the HTML or the CSS file. 💯️